docs/contributing/how-to-release.md

```diff
@@ -117,8 +117,12 @@ Create https://docs.wxwidgets.org/x.y.z/ (ask Bryan to do it if not done yet).
 
 ## Announcement
 
-Update https://www.wxwidgets.org, usually a news item is enough but something
-more can be called for for major releases.
+Update https://www.wxwidgets.org:
+* Update release information (at least `version` and `released`) in `_data/relases.yml`.
+* Download information can than be updated by running `update_release_info.rb`.
+  This will update the asset information from github.
+* Add a news item. Usually a news item is enough but something
+  more can be called for for major releases
 
 Post `docs/publicity/announce.txt` at least to wx-announce@googlegroups.com and
 to wx-users for the important releases.
```

Comment:
Considering the frequency of releases, I think all releases in both branches could be announced in wx-users.